### PR TITLE
Provide overview of implementations by partners

### PR DESCRIPTION
**Pull request by tomkerkhove**
Live preview: https://oam-fork.netlify.com/

It adds sections per major vendor with their logo, similar to Rudr, other implementations are grouped below:
image

Relates to https://github.com/oam-dev/spec/issues/295

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Approach one:
image

Alternatively we can add sections per (major) vendor with their logo similar to Rudr, but that can be overkill.

**Comment by tomkerkhove** [comment]
Happy to help @suhuruli

File: public/index.html [hunk 1]
```diff
@@ -324,6 +324,51 @@ <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
...
+    </div>
+
+    <div class="columns">
+      <div class="column is-quarter has-text-left">
+      </div>
+      <div class="column is-three-quarters">
+
+        <p>Alibaba supports Open Application Model in their offerings:</p>
+        <ul>
+          <li><b>Alibaba Enterprise Distributed Application Service</b> (Available) - A PaaS platform for a variety of application deployment options and microservices solutions.</li>
```
Comment (wonderflow) [inline public/index.html]: can you help add a link here  https://www.alibabacloud.com/product/edas

File: public/index.html [hunk 2]
```diff
@@ -324,6 +324,51 @@ <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
...
+          <li><b>Alibaba Resource Orchestration Service</b> (Available) - Provides developers and system managers with a simple method to create and manage their Alibaba Cloud resources.</li>
```
Comment (wonderflow) [inline public/index.html]: https://www.alibabacloud.com/product/ros

We'll open source this project soon, we could update this link until then

**Comment by tomkerkhove** [comment]
Thanks @wonderflow!

Added a "More info" at the end:
image

Unless you want me to move the link somewhere else?

File: public/index.html (hunk 2 above)
Comment (tomkerkhove) [reply public/index.html]: Done, see suggestion below

File: public/index.html (hunk 1 above)
Comment (tomkerkhove) [reply public/index.html]: Done, see suggestion below

**Comment by wonderflow** [comment]
@tomkerkhove it's cool

**Comment by tomkerkhove** [comment]
Good with this approach @prasek @negz?

**Comment by tomkerkhove** [comment]
`Upcoming Implementations` is a good suggestion!

Shall we leave out Crossplane then and move forward for the rest?

**Comment by tomkerkhove** [comment]
Done, thanks for the review @resouer !

**Comment by resouer** [comment]
LGTM

**Comment by tomkerkhove** [comment]
Thanks @resouer - What is the typical deployment frequency for this repo?

**Comment by resouer** [comment]
I'm really not sure, @suhuruli should know more.

**Comment by suhuruli** [comment]
@flynnduism , can we deploy a new version of this website with these changes?

**Comment by flynnduism** [comment]
@suhuruli yes I will redeploy this site now

 The [README has the deployment info](https://github.com/oam-dev/oam-dev.github.io#deployment) FYI. Github Pages _does not_ auto-deploy the Hugo site when PRs are merged, but with some extra work we could make it so.

**Comment by flynnduism** [comment]
Site redployed now @suhuruli - https://oam.dev/

**Comment by tomkerkhove** [comment]
Thank you!

Noticed the logos are not good on mobile but opened an issue for it (#24).

Sorry about that.

**Pull request merged**